<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ct3AddModifyDialog</class>
 <widget class="QDialog" name="ct3AddModifyDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>367</width>
    <height>280</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ct3AddModifyDialog</string>
  </property>
  <property name="whatsThis">
   <string/>
  </property>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>17</y>
     <width>54</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>54</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>54</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="text">
    <string>配置名</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="templateNameEdit">
   <property name="geometry">
    <rect>
     <x>102</x>
     <y>18</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QListWidget" name="ct3LayerPosListWidget">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>60</y>
     <width>100</width>
     <height>130</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>130</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>130</height>
    </size>
   </property>
   <property name="sortingEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_53">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>45</y>
     <width>54</width>
     <height>21</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>54</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>54</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="text">
    <string>等层距/多层</string>
   </property>
  </widget>
  <widget class="QComboBox" name="ct3MultiLayerComboBox">
   <property name="geometry">
    <rect>
     <x>102</x>
     <y>45</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>80</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>80</width>
     <height>16777215</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>20</y>
     <width>54</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>54</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>54</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="text">
    <string>断层位置</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>72</y>
     <width>54</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>54</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>54</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="text">
    <string>图像矩阵</string>
   </property>
  </widget>
  <widget class="QComboBox" name="ct3MatrixComboBox">
   <property name="geometry">
    <rect>
     <x>102</x>
     <y>72</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>80</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>80</width>
     <height>16777215</height>
    </size>
   </property>
  </widget>
  <widget class="QLineEdit" name="ct3LayerPosEdit">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>20</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>99</y>
     <width>54</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>54</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>54</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="text">
    <string>视场直径</string>
   </property>
  </widget>
  <widget class="QComboBox" name="ct3ViewComboBox">
   <property name="geometry">
    <rect>
     <x>102</x>
     <y>99</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>80</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>80</width>
     <height>16777215</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>126</y>
     <width>54</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>54</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>54</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="text">
    <string>采样时间</string>
   </property>
  </widget>
  <widget class="QComboBox" name="ct3SampleTimeComboBox">
   <property name="geometry">
    <rect>
     <x>102</x>
     <y>126</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>80</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>80</width>
     <height>16777215</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>153</y>
     <width>54</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>54</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>54</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="text">
    <string>扫描方位</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ct3AngleLineEdit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>150</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="ct3LayerSpaceLabel">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>179</y>
     <width>54</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>54</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>54</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="text">
    <string>层间距</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ct3LayerSpaceLineEdit">
   <property name="geometry">
    <rect>
     <x>102</x>
     <y>179</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>80</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="ct3SpaceNumLabel">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>205</y>
     <width>54</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>54</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>54</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="text">
    <string>层数</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ct3LayerNumLineEdit">
   <property name="geometry">
    <rect>
     <x>102</x>
     <y>205</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>80</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>80</width>
     <height>16777215</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>230</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>230</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
